src/main.rs

```diff
@@ -1082,6 +1096,56 @@ mod tests {
         });
     }
 
+    #[test]
+    fn test_vunet() {
```

Comment:
`test_vhost_user_net`, please, to make it readable and clearer.

Reply:
change done!